src/libs/actions/BankAccounts.js

```diff
@@ -821,6 +829,59 @@ function updateReimbursementAccountDraft(bankAccountData) {
 }
 
 /**
+<<<<<<< HEAD
+ * Triggers a modal to open allowing the user to reset their bank account
+ */
+function requestResetFreePlanBankAccount() {
+    Onyx.merge(ONYXKEYS.REIMBURSEMENT_ACCOUNT, {shouldShowResetModal: true});
+}
+
+/**
+ * Hides modal allowing the user to reset their bank account
+ */
+function cancelResetFreePlanBankAccount() {
+    Onyx.merge(ONYXKEYS.REIMBURSEMENT_ACCOUNT, {shouldShowResetModal: false});
+}
+
+/**
+ * Reset user's reimbursement account. This will delete the bank account.
+ *
+ * @param {String} password
+ */
+function resetFreePlanBankAccount(password = 'Password1') {
+    const bankAccountID = lodashGet(reimbursementAccountInSetup, 'bankAccountID');
+    if (!bankAccountID) {
+        throw new Error('Missing bankAccountID when attempting to reset free plan bank account');
+    }
+    if (!credentials || !credentials.login) {
+        throw new Error('Missing credentials when attempting to reset free plan bank account');
+    }
+    if (!password) {
```

Comment:
Can we reach this, if we default the password to `Password1`?

Reply:
This can also be removed.